.rubocop.yml

```diff
@@ -24,6 +24,10 @@ Lint/Debugger:
   DebuggerMethods:
     Pry: ~
 
+# Need to allow reloading const.rb from fixup.rb
+Lint/OrAssignmentToConstant:
```

Comment:
If we end up needing to use conditional assignment in const.rb, then disable this rule for that file only, rather than the entire codebase.

Reply:
Good idea.